modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/tests

```diff
@@ -15,15 +63,27 @@
     exodiff = 'poly_2d_back_only_nssb.e'
     exodiff_opts = '-pedantic'
     cli_args = 'Mesh/gen/background_block_ids="30 35" Mesh/gen/background_block_names="back_in back_out" Mesh/gen/generate_side_specific_boundaries=false --mesh-only "poly_2d_back_only_nssb.e"'
-    requirement ='The system shall generate a hexagon mesh without rings and ducts and without generating side specific boundaries.'
+    requirement = 'The system shall generate a hexagon mesh without rings and ducts and without generating side specific boundaries.'
     recover = false
   []
   [back_only_single_block]
     type = 'Exodiff'
     input = 'poly_2d.i'
     exodiff = 'poly_2d_back_only_sb.e'
     cli_args = 'Mesh/gen/quad_center_elements=true Mesh/gen/background_block_ids="30" Mesh/gen/background_block_names="back" --mesh-only "poly_2d_back_only_sb.e"'
-    requirement ='The system shall generate a polygon mesh without rings and ducts and with a multilayer QUAD background using a single set of block id and name.'
+    requirement = 'The system shall generate a polygon mesh without rings and ducts and with a multilayer QUAD background using a single set of block id and name.'
+    recover = false
+  []
+  [back_only_single_block_quadratic_tri7_quad9]
```

Comment:
This doesn't actually generate any TRI7, does it?  If not then the name is a bit misleading.

Reply:
Right, I changed requirement but forgot to change the name.